src/Controls/tests/UITests/Tests/Issues/Issue22000.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue22000 : _IssuesUITest
+	{
+		public Issue22000(TestDevice device) : base(device) { }
+
+		public override string Issue => "[Windows] Carousel changes the current position when window is resized";
+
+		[Test]
+		[Category(UITestCategories.CarouselView)]
+		public void ResizeCarouselViewKeepsIndex()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.iOS, TestDevice.Mac });
```

Comment:
Is this test relevant on all platforms?
Wondering if we should run for everyone?

Reply:
Yeah, could be interesting for all the platforms. Applied changes.